spec/rubocop/cop/rspec/predicate_matcher_spec.rb

```diff
@@ -64,6 +66,7 @@
           expect(foo).to include(something)
           expect(foo).to respond_to(:bar)
           expect(foo).to be_something()
+          expect(foo).to match_style(bar: 'baz')
```

Comment:
Would this work with any method?
`rspec-expectations` magically provides `have_` and `be_` dynamic predicate matchers, but I suspect not `match_` ones.
In this case, it's an existing matcher defined by capybara. What if such a matcher doesn't exist, e.g.:

```ruby
expect(inventory.matches_or_a_lighter?).to be_truthy
```

Reply:
I remembered the plan that from now on Capybara related cops will be separated from RuboCop RSpec, so I thought again in the direction of creating a new cop in `RSpec/Capybara/*`. What do you think?

Reply:
Makes sense :+1: